src/common/PeopleTile/styled.js

```diff
@@ -32,7 +32,7 @@ export const StyledTile = styled.div`
           "poster personDetails"
           "personDescription personDescription";
         grid-template-columns: 124px 1fr;
-        width: 290px;
+        width: auto;
```

Comment:
Rozumiem tutaj twój zamysł, jak najbardziej jest to sensowne, ale subiektywnie mnie się bardziej te komponenty podobają mimo wszystko dłuższe, ale takie bardziej pionowe, ale to już jest kwestia jak komu pasuje, nie upieram się przy jednej wersji, jeśli mnie przegłosujecie to możemy wprowadzić taką poprawkę 😂

Reply:
@TSypko a Tobie jak się bardziej podoba?  Twoje zdanie będzie decydujące 😅

Reply:
Myślałem, że komentarz w review, że jestem za zmianami to wyjaśnia, ale potwierdzam - wolę auto :) 
Dla mnie ekran w mobilkach jest na tyle mały, że nie warto jeszcze go dodatkowo ograniczać.  Można by ewentualnie wykorzystać breakpoint headerMax i w stanie między mobile a headerMax gdzie jest więcej miejsca dać jakąś szerokość na sztywno zgodnie z wizją @mateusz24polok .